pxtsim/runtime.ts

```diff
@@ -610,13 +611,17 @@ namespace pxsim {
             function buildResume(s: StackFrame, retPC: number) {
                 if (currResume) oops("already has resume")
                 s.pc = retPC;
+                let start = Date.now()
```

Comment:
Util.now()?

Reply:
We seem to be using `Date.now` much more than `Util.now`